.gitignore

```diff
@@ -1,3 +1,4 @@
 node_modules
 .DS_Store
-npm-debug.log
\ No newline at end of file
+npm-debug.log
+.nyc_output
```

Comment:
Where does this file type come from?

Reply:
The npm package, tap, which I used for testing and coverage generates this directory with coverage information

Reply:
Cool. I've been wanting to check out tap. The output looks good.